package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "loopback-boot",
-  "version": "2.24.1",
+  "version": "2.24.2",
```

Comment:
Please revert. We are incrementing package version as part of our release process.